atomic_reactor/plugins/prepub_flatpak_create_oci.py

```diff
@@ -64,41 +64,45 @@ class FlatpakCreateOciPlugin(PrePublishPlugin):
     key = 'flatpak_create_oci'
     is_allowed_to_fail = False
 
-    def __init__(self, tasker, workflow):
+    def __init__(self, workflow):
         """
-        :param tasker: ContainerTasker instance
         :param workflow: DockerBuildWorkflow instance
         """
-        super(FlatpakCreateOciPlugin, self).__init__(tasker, workflow)
+        super(FlatpakCreateOciPlugin, self).__init__(workflow)
         self.builder = None
 
         try:
             self.flatpak_metadata = self.workflow.conf.flatpak_metadata
         except KeyError:
             self.flatpak_metadata = FLATPAK_METADATA_ANNOTATIONS
 
-    def _export_container(self, container_id):
-        export_generator = self.tasker.export_container(container_id)
-        export_stream = StreamAdapter(export_generator)
-
-        outfile, manifestfile = self.builder._export_from_stream(export_stream)
-
-        return outfile, manifestfile
+#    def _export_container(self, container_id):
+#        export_generator = self.tasker.export_container(container_id)
+#        export_stream = StreamAdapter(export_generator)
+#
+#        outfile, manifestfile = self.builder._export_from_stream(export_stream)
+#
+#        return outfile, manifestfile
 
     def _export_filesystem(self):
-        image = self.workflow.image
+        # image = self.workflow.image
         self.log.info("Creating temporary docker container")
         # The command here isn't used, since we only use the container for export,
         # but (in some circumstances) the docker daemon will error out if no
         # command is specified.
-        container_dict = self.tasker.create_container(image, command=["/bin/bash"])
-        container_id = container_dict['Id']
```

Comment:
The comment is not relevant anymore

Reply:
the code is still there just commented out, so I've left comments there

Reply:
ah I see it now